cohttp-async/src/server.mli

```diff
@@ -104,3 +104,20 @@ val create :
   response Async_kernel.Deferred.t) ->
   ('address, 'listening_on) t Async_kernel.Deferred.t
 (** Build a HTTP server, based on the [Tcp.Server] interface *)
+
+val create_tcp_handler :
+  'addr ->
+  Async_unix.Reader.t ->
+  Async_unix.Writer.t ->
+  (body:Body.t -> 'addr -> Http.Request.t -> response Async_kernel.Deferred.t) ->
+  unit Async_kernel.Deferred.t
+
+val create_expert_tcp_handler :
+  'addr ->
+  Async_unix.Reader.t ->
+  Async_unix.Writer.t ->
+  (body:Body.t ->
+  'addr ->
+  Http.Request.t ->
+  response_action Async_kernel.Deferred.t) ->
+  unit Async_kernel.Deferred.t
```

Comment:
I  think our convention elsewhere is to add an `Expert` submodule.

Btw, some minor docstrings for these functions would be appreciated.

Reply:
I like the idea of using `Expert` here. I've switched to using that. Also added some basic docs about why these functions exist, and who might find it useful.